src/vs/workbench/services/auxiliaryWindow/browser/auxiliaryWindowService.ts

```diff
@@ -3,28 +3,28 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { localize } from 'vs/nls';
-import { mark } from 'vs/base/common/performance';
-import { Emitter, Event } from 'vs/base/common/event';
+import { getZoomLevel } from 'vs/base/browser/browser';
```

Comment:
@justschen this change of imports seems not required for this task, do you have some kind of setting enabled to reshuffle imports?